test/src/org/labkey/test/components/targetedms/QCSummaryWebPart.java

```diff
@@ -176,6 +177,10 @@ private void readSummary()
             Pattern precursorPattern = Pattern.compile("(\\d+) precursor");
             Matcher precursorMatcher = precursorPattern.matcher(qcSummary);
             _precursorCount = precursorMatcher.find() ? Integer.parseInt(precursorMatcher.group(1)) : 0;
+
+            Pattern metricPattern = Pattern.compile("(\\d+) metric");
```

Comment:
This should be fine, but I'm always happy to tag the DOM with IDs or classes to help tests find the right parts.

Reply:
This is seems to be okay for now. Thank you